network/yarn/pom.xml

```diff
@@ -96,6 +107,54 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
regression control. How do you think I got it to work in the first place? How to verify that it keeps working? I want to add some more shading; for that I'll use a small ant macro to make things terser. 

Consider this: A build is an execution of a program; programs needs tests. This is the test of the shading process.